input_parser.star

```diff
@@ -141,8 +144,25 @@ DEFAULT_L1_ARGS = {
     # Number of seconds per slot on the Beacon chain
     # Default: 12
     "l1_seconds_per_slot": 1,
-    # The amount of ETH sent to the admin, sequence, aggregator and sequencer addresses.
-    "l1_funding_amount": "100ether",
+    # The amount of ETH sent to the admin, sequence, aggregator, sequencer and other chosen addresses.
+    "l1_funding_amount": "1000000ether",
+    # Default: 2
+    "l1_participants_count": 2,
```

Comment:
So now we have 2 EL + CL instead one one, I think we should keep the default to 1 EL + CL. If the `assertor` is specified in the additional services, we can override this value and set it to 2 for example.